bundles/org.openhab.persistence.dynamodb/src/test/java/org/openhab/persistence/dynamodb/internal/BaseIntegrationTest.java

```diff
@@ -137,8 +124,7 @@ private static int findFreeTCPPort() {
     protected void waitForAssert(Runnable runnable) {
         // Longer timeouts and slower polling with real dynamodb
         // Non-CI tests against local server are with lower timeout.
-        waitForAssert(runnable, hasFakeServer() ? isRunningInCI() ? 30_000L : 10_000L : 120_000L,
-                hasFakeServer() ? 500 : 1000L);
+        waitForAssert(runnable, hasFakeServer() ? 30_000L : 120_000L, hasFakeServer() ? 500 : 1000L);
```

Comment:
Can you adapt comment above as well to avoid confusion?

Reply:
I think the new one makes more sense.